tests/contract/test_resource_client.py

```diff
@@ -1210,10 +1226,12 @@ def test_generate_invalid_update_example_with_inputs(resource_client_inputs):
 
 def test_generate_update_example_with_primary_identifier(resource_client_inputs_schema):
     created_resource = resource_client_inputs_schema.generate_create_example()
```

Comment:
recommending to test locally by running contract test against a resource.

Reply:
updated the contract test updated in the details section